cli/worker.rs

```diff
@@ -128,10 +115,32 @@ impl Worker {
       waker: AtomicWaker::new(),
       internal_channels,
       external_channels,
-      inspector,
+      inspector: None,
     }
   }
 
+  pub fn attach_inspector(&mut self) {
+    let s = self.state.borrow();
+    let global_state = s.global_state.clone();
```

Comment:
Why does it need to be cloned?

Reply:
@piscisaureus ignore for now; this is a bit more complex problem that I have idea for; I'll try in the morning

Reply:
Took the liberty of removing it myself and pushing it to your branch.